crates/extensions/c8y_mapper_ext/src/service_monitor.rs

```diff
@@ -28,6 +28,8 @@ pub fn convert_health_status_message(
     let mut mqtt_messages: Vec<Message> = Vec::new();
 
     // If not Bridge health status
+    // FIXME: can also match "device/bridge//" or "/device/main/service/my_custom_bridge"
```

Comment:
That looks like a very fragile logic. Creating an issue would be nice, otherwise these FIXMEs might get forgotten/lost, unless you had some plans to fix it immediatley.

Reply:
I'll make a separate issue and a PR for this, for visibility.

Reply:
#2370